projects/plugins/jetpack/modules/carousel/jetpack-carousel.php

```diff
@@ -239,6 +239,14 @@ function enqueue_assets() {
 				true
 			);
 
+			$swiper_library_path = array(
+				'url' => Assets::get_file_url_for_environment(
+					'_inc/build/carousel/swiper-bundle.min.js',
+					'modules/carousel/swiper-bundle.js'
+				),
+			);
+			wp_localize_script( 'jetpack-carousel', 'jetpackSwiperLibraryPath', $swiper_library_path );
```

Comment:
Since we do not enqueue the file directly, will this work with plugins that concatenate JS, like Autoptimize (enabled on some WoA sites)?

Reply:
I may be misunderstanding what you are saying here, but the `wp_localize_script` is running against the existing the existing `jetpack-carousel` script which is enqueued?

Reply:
FYI, I ran Autoptimize locally with the js and css options toggled on and everything still worked as expected.